src/Components/Subscriptions/Table/index.jsx

```diff
@@ -136,12 +143,20 @@ const Table = ({ data }) => {
         </div>
       )}
       {showConfirmDeleteModal && (
-        <ConfirmDeleteModal
-          onClose={() => setShowConfirmDeleteModal(false)}
-          confirmDelete={() => handleDelete(editingSubscriptionId)}
-        />
+        <ConfirmModal
+          title="Delete Subscription"
+          handler={closeModal}
+          onAction={() => handleDelete(editingSubscriptionId)}
+          reason="delete"
+        >
+          <p>Are you sure to delete subscription?</p>
```

Comment:
In the definition of the confirmModal, the children is actually a paragraph tag. You can put text directly.